src/networks/osmotest5.ts

```diff
@@ -24,8 +24,14 @@ export class OsmoTest5 extends Network {
       contractRegistry: contractRegistry,
       poolRegistry: poolRegistry,
       exchanges: [
+        // new OsmosisDex({
+        //   poolUrl:
+        //     'https://lcd.testnet.osmosis.zone/osmosis/gamm/v1beta1/pools?pagination.limit=1000',
+        //   volumeUrl: undefined,
+        // }),
         new OsmosisDex({
-          poolUrl: 'https://lcd.testnet.osmosis.zone/osmosis/gamm/v1beta1/pools?pagination.limit=1000',
+          poolUrl:
+            'https://lcd.testnet.osmosis.zone/osmosis/concentratedliquidity/v1beta1/pools?pagination.limit=1000',
```

Comment:
We should create a new `gammPoolUrl` option for the dex and add them based on this. We shouldn't be combining this logic with the existing since they're literally separate APIs.

Reply:
So gammPoolUrl and concentratedPoolUrl, then fetch and parse them separately?

Reply:
Yeah especially if the api responses are a different shape, then separation would be good. Especially if the logic paths are different.